Makefile

```diff
@@ -72,15 +72,15 @@ create_qa:
 		--role arn:aws:iam::946183545209:role/lambda-full-access \
 		--zip-file fileb://./deploy/rhrc.zip \
 		--profile qa
-		
+
 deploy_qa: ## Deploys the latest version to AWS QA
 	@cp config/var_qa.env config/var_deploy.env
 	@export AWS_DEFAULT_PROFILE=qa
 	@make package
 	aws lambda update-function-code \
 		--function-name RecapHoldRequestConsumer-qa \
 		--zip-file fileb://./deploy/rhrc.zip \
-		--profile qa
```

Comment:
Can you create a profile locally instead of changing this? OR... make it consistent with the create script?

Reply:
I think most people have nypl-digital-dev profiles. If it's the same to you I'll change the create script to be consistent

Reply:
The profile is configured locally (for me is just another entry I created at `~/.aws/credentials`). You can set it up however you like, right? The concern here is that you don't want to confuse qa and production at the point of deployment, so I separated them. nypl-digital-dev is fine. I just prefer keeping them separate for peace of mind.